File: src/test/java/org/openrewrite/python/tree/AssertTest.java
package org.openrewrite.python.tree;

import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.ValueSource;
import org.openrewrite.test.RewriteTest;

import static org.openrewrite.python.Assertions.python;

public class AssertTest implements RewriteTest {

    @ParameterizedTest
    @ValueSource(strings = {
      "assert x",
      "assert  x",
      "assert x, y",
      "assert  x, y",
      "assert x , y",
      "assert x,  y",
    })
    public void assert_(String arg) {
        rewriteRun(python(arg));
    }

}

File: rewrite/tests/python/assert_test.py
from rewrite.test import rewrite_run, python


def test_formatting():
    # language=python
    rewrite_run(
        python(
            """\
            assert \\
            True \\
            ,\\
            'foo'
            """,
        ),
    )


def test_with_message():
    # language=python
    rewrite_run(python("assert True, 'foo'"))


def test_assert():
    # language=python
    rewrite_run(python("assert True"))
